src/main/java/sage/tasktypes/Task.java

```diff
@@ -0,0 +1,29 @@
+package sage.tasktypes;
+
+public class Task {
+    private String taskName = "";
+    private boolean completed = false;
```

Comment:
Your boolean should sound more like a boolean so that you do not need another function
```suggestion
    private boolean isCompleted = false;
```